app/src/main/java/com/brainwallet/navigation/LegacyNavigation.kt

```diff
@@ -24,8 +23,9 @@ object LegacyNavigation {
         auth: Boolean
     ) {
         Timber.i("timber: startBreadActivity: %s", from.javaClass.name)
-        val toStart: Class<*> = if (auth) LoginActivity::class.java else BreadActivity::class.java
-        val intent = Intent(from, toStart)
+        val intent = if (auth) BrainwalletActivity.createIntent(from, Route.UnLock)
+        else Intent(from, BreadActivity::class.java)
+
```

Comment:
Im certain this is WIP @andhikayuana but I was unable to unlock around this commit and I suspect there is something being worked on.
![cantunlock](https://github.com/user-attachments/assets/bca549dc-41f1-463f-9975-ef9802b1613d)

Reply:
I just switch back to the previous one, will take a look later

I am going to offline at this moment

Reply:
I just pushed here https://github.com/brainwallet-co/android/pull/23/commits/29a89f2d0de33112e331af9c654f5252a9b23add for this issue, let me know if you are still having this issue 
Thanks!